### PR TITLE
propose using Issues for discussing RFCs after the process has finished

### PR DESCRIPTION
**Pull request by cybojenix**

### CODE REVIEW

**Comment by cybojenix** [comment]
We'll also want to introduce Issue Templates to give guidance on how to raise an issue?

**Comment by kjdchapman** [comment]
I'm not sure if RFCs are working full stop. 
- It's not clear what needs to happen for one to be approved
- Can everyone find the time to engage with them?
- Are written discussions a suitable replacement for (higher-bandwidth) in-person discussions, especially around sensitive/difficult topics?
- Are they suitable for a company that's growing to encompass more people from a non-development background?

I like the intent behind this - that culture is an ongoing discussion rather than rules that are made and then last forever - but I worry the whole issues/rfcs/github repo for culture might not be the right road.

**Comment by davidwinter** [comment]
This change has my vote. 

@kjdchapman you raise a good point, but feel like it's a separate and very important discussion to be had elsewhere perhaps? Perhaps a comradro topic?

**Comment by kjdchapman** [comment]
Agreed @davidwinter - I'd already put a rough card in there. My comment above is me expressing worry about effort on incremental changes. I think we should step back and rework things - I guess I should get on with that.

Zooming in a bit more, I'm happy with this particular change.

**Comment by cybojenix** [comment]
Given that this is driven from the LS side, I'll wait for more approvals before merging this

**Comment by craigjbass** [comment]
Which RFCs do we see not adopted?

File: README.md [hunk 1]
```diff
@@ -25,8 +25,16 @@ Standards RFCs shouldn’t be substantially altered after they are accepted, alt
 3. When the new RFC is accepted and the PR is merged, the old RFC will no longer be active.
 
 ## Managing Action Plans
+
 For RFCs where the outcome is an agreed Action Plan, you may want to update the RFC with meaningful status updates in new PRs. Once the plan is either complete or no longer relevant, it should be moved to the archived directory in a new PR.
 
+## Raising a concern with an RFC
+
+There is no definitive process for discussing an RFC before the above process has been started, or after it has finished. To faciliate this, an [Issue][Issues] may be created to discuss potential changes.
```
Comment (craigjbass) [inline README.md]: My understanding is that new RFCs can be opened "Request for Comment" that can obsolete previous RFCs to update the wording to match current behaviour

**Comment by cob16** [comment]
> Which RFCs do we see not adopted?

rfc-012-makefile-standards.md for example anything past setup/serve

**Comment by davidwinter** [comment]
> > Which RFCs do we see not adopted?
> 
> rfc-012-makefile-standards.md for example anything past setup/serve

The point of the RFCs (at least my understanding) were for them to be followed as a standard by the company, unless there's a good reason for not doing so, which should result in the RFC being amended to cater for those use cases. 

The Live Services team have found a good number of our projects not conforming to the RFC's, a good example being RFC012 (Makefile's), and if they're not being used or followed, then the question should be asked, why, and is there any point having them, or investing the time in writing them. And I think we then get back to the issues that @kjdchapman mentioned, and that should perhaps be a separate issue or comradrospective.

File: README.md (hunk 1 above)
Comment (davidwinter) [reply README.md]: Unless we open blank PRs, the reason for suggesting Issues was to gauge opinion on a change to an RFC, or even a new RFC, before investing the time in writing one.

**Pull request merged**